dashboard/memory_utils.py

```diff
@@ -340,7 +340,11 @@ def track_reference_size(group):
         "ACTOR_HANDLE": "total_actor_handles"
     }
     for entry in group["entries"]:
-        d[table_name[entry["reference_type"]]] += entry["object_size"]
+        size = entry["object_size"]
```

Comment:
This was another bug that is unrelated to this PR. This makes the size of actor handle negative.